railties/lib/rails/application/configuration.rb

```diff
@@ -79,6 +79,7 @@ def load_defaults(target_version)
 
           if respond_to?(:action_view)
             action_view.form_with_generates_remote_forms = true
+            action_view.form_with_skip_default_ids = false
```

Comment:
Aren't we? It was a conscious decision to skip them in 5.1 at the time, *but* if we're saying that change introduced a bug (and doesn't have any worse effect than making HTML slightly bulkier) ISTM we might reasonably backport the do-use-ids behaviour to fix the bug.